tests/scripts/github_docs_comment.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import logging
+import argparse
+import sys
+from urllib import error
+
+from git_utils import git, GitHubRepo, parse_remote
+from cmd_utils import init_log
+
+
+def build_docs_url(base_url_docs, pr_number, build_number):
+    return f"{base_url_docs}/tvm/PR-{str(pr_number)}/{str(build_number)}/docs/index.html"
```

Comment:
```suggestion
    return f"{base_url_docs}/PR-{str(pr_number)}/{str(build_number)}/docs/index.html"
```

the new URL also removes the `tvm` prefix